libs/acn/LLRPProbeRequestPDUTest.cpp

```diff
@@ -191,6 +191,15 @@ void LLRPProbeRequestPDUTest::testPrepend() {
     0x56, 0x78, 0, 0, 0, 2,
   };
   OLA_ASSERT_DATA_EQUALS(expected_data, sizeof(expected_data), buffer, length);
+
+  // test null stack
+  LLRPProbeRequestPDU::PrependPDU(NULL,
```

Comment:
does this test actually test something? shouldn't it check the return value?

Reply:
As above